### PR TITLE
Add typings

### LINKED ISSUE #40: Include typings
### I'm submitting a...
- [ ] Bug report
- [x] Feature request

The typings from [here](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/5344bfc80508c53a23dae37b860fb0c905ff7b24/types/simplemde) should be adapted for EasyMDE and included in this repo.

### PR DESCRIPTION
**Pull request by FranklinWhale**
Fixes #40 

To test it, create a Node.js project, then use `npm install github:FranklinWhale/easy-markdown-editor#add-typings` to use my branch

### CODE REVIEW

File: types/easymde.d.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        renderingConfig?: RenderingOptions;
+        shortcuts?: Shortcuts;
+        showIcons?: ReadonlyArray<string>;
+        spellChecker?: boolean;
+        status?: boolean|ReadonlyArray<string|StatusBarItem>;
+        styleSelectedText?: boolean;
+        tabSize?: number;
+        toolbar?: boolean|ReadonlyArray<string|ToolbarIcon>;
+        toolbarTips?: boolean;
+    }
```
[the remark this answers is not in the document]
Comment (FranklinWhale) [reply types/easymde.d.ts]: `onToggleFullScreen?` has been added. Thanks.

Regarding testing the typings by Travis, would you add `npm run test:types` to Travis script?

**Comment by FranklinWhale** [comment]
Regarding testing the typings by Travis, would you add `npm run test:types` to Travis script?

**Comment by FranklinWhale** [comment]
@Ionaru Thanks!

**Comment by GenaBitu** [comment]
Hi, I was just gonna ask if you provide TS support and you've just added it! Thanks a lot.

**Pull request merged**